src/classes/TDTM_ObjectDataGateway_TEST.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>43.0</apiVersion>
```

Comment:
This can be api v46.

Reply:
Changed version to 46. It was 43 initially since `TDTM_ObjectDataGateway` is v43.